projects/storefrontlib/src/shared/components/list-navigation/pagination/pagination.component.ts

```diff
@@ -16,6 +19,79 @@ export class PaginationComponent {
   @Input() pagination: PaginationModel;
   @Output() viewPageEvent: EventEmitter<number> = new EventEmitter<number>();
 
+  getPagePrevious(): number {
+    return this.pagination.currentPage;
+  }
+
+  getPageNext(): number {
+    return this.pagination.currentPage + 2;
+  }
+
+  getPageIndicies(): Array<number> {
```

Comment:
does it really need to be an array with one number in it?

Reply:
Needs to be an array the size of the total pages in order to generate the page index using ngFor.

Reply:
Other option could be to use an array of [0, 1, 2, ... ] and use those as i instead of `let i = index;`

Reply:
if that's needed, well, let it be :)